iphone/metadata/pl/description.txt

```diff
@@ -0,0 +1,38 @@
+• Szczegółowe mapy offline z miejscami, które nie istnieją na innych mapach
+• Trasy rowerowe, szlaki turystyczne, ścieżki spacerowe
+• Linie konturowe, profile wysokościowe, szczyty i zbocza
+• Nawigacja piesza, rowerowa i samochodowa typu "turn-by-turn" ze wskazówkami głosowymi
+• Szybkie wyszukiwanie w trybie offline
+• Zapisz wszystko czego potrzebujesz, importuj i udostępniaj KML/KMZ (planowany jest GPX)
+• Tryb ciemny, aby chronić Twoje oczy
+
+Organic Maps jest <b>czyste i organiczne, wykonane z miłością</b>:
+
+• Szanuje Twoją prywatność
+• Oszczędza baterię
+• Brak niespodziewanych opłat za transmisję danych w sieci komórkowej
+
+Wolny od elementów śledzących i innych złych rzeczy:
+
+• Brak reklam
+• Brak śledzenia
+• Brak zbierania danych
+• Brak telemetrii
+• Brak irytującej rejestracji
+• Brak obowiązkowych samouczków
+• Brak hałaśliwego spamu e-mailowego
+• Żadnych powiadomień typu push
+• Brak 💩 oprogramowania
```

Comment:
Do we really want it in our description? :)

Reply:
let's remove this emoji and merge PR.